src/myfunc.cc

```diff
@@ -8,7 +8,8 @@
 void
 myfunc3(void)
 {
-	spdlog::info("Malloc in lib: \n");
+	int a;
+	spdlog::info("Malloc in lib: \n", a);
```

Comment:
variable 'a' is uninitialized when passed as a const reference argument here [clang-diagnostic-uninitialized-const-reference]